docs/segments/manage_segments.rst

```diff
@@ -3,4 +3,493 @@
+
+3. Click the **Filters** tab to add filters.
+
+4. Click the **Choose one**… menu and search for the field you’d like to Segment by. Listed below are three types of fields:
+      
+  * Contact fields
+  
+    * Set Fields to **Available for Segments = Yes** in your Custom Field manager to display here.
+  * Contact behavior and actions
+  * Primary Company fields
+    * Set Fields to **Available for Segments = Yes** in your Custom Field manager to appear here.
+    * Contacts associates with multiple Companies, but Mautic adds them to Segments based on fields for the primary Company.
+
+5. Add more filters, using the **And** and **Or** operators. An **Or** operator creates a new group of filters which can include And operators.
+
+6. Click **Save and close**.
+
+.. note:: 
+
+    The following filters rebuild immediately in real time, while other filters rebuild with the nightly refresh:
+
+    * All core fields
+    * Custom fields added by the User
+    * Category
+    * Tags
+    * Contact Email received
+    * Hit URL
+    * Owner ID
+
+    * The nightly Segment rebuild jobs for each Segment are randomized for six hours. Jobs are executed anytime between midnight to 6 AM with respect to the **Default timezone** set for the instance in **Settings > Configurations > System Settings**.
+    * If a Segment fails to rebuild for a predetermined length of time, Mautic displays a notification alerting you of an error. For information on defining this time period, see Segment settings 
+    * The include/exclude operators with pipe-separated values are supported in the Text field types in Segment filters. The values can also be pasted from the spreadsheet.
+
+.. vale off
+
+Using Date Filters
+==================
+
+.. vale on
+
+You can create dynamic Segments by using date filters.
+
+.. note:: 
+  
+  The date format for values stored in the database is ``YYYY-MM-DD``. For example, December 11, 2020 is stored as 2020-12-11 and November 12, 2020 is stored as 2020-11-12. To update the display format for dates, go to **Settings > Configuration > System Settings > System Defaults**. However, this doesn’t alter the format in which dates are stored in the database.
+
+.. vale off
+
+Operators
+=========
+
+.. image:: images/operators.png
+    :alt: Screenshot showing Operators.
+
+.. vale on
+
+You must ensure that you use the correct operator and time frame to build an effective Segment.
+
+Once you’ve selected a date field as your filter, such as the default **Date last active** field or a custom **Birthday** field, you’ll have a list of operators to choose from:
+
+* **Equals** - The value on the Contact record exactly matches the filter value.
+
+* **Not Equal** - The value on the Contact record is any value that doesn’t match the filter value.
+
+* **Greater than** - The value on the Contact record is at a later date in time than X date. For example, **Greater than** today means anytime from tomorrow until the end of time.
```

Comment:
In the front end it is a tiny bit confusing having the second bit of text in bold, especially when it lines up with the bold texts at the start of the line. I might suggest seeing how it looks using an inline code markup rather than bold, perhaps. See what you think!